pages/incometax.vue

```diff
@@ -75,15 +75,27 @@
       <section-title path="incometax.howto.title" />
       <paragraph path="incometax.howto.p1" :margin-bottom="false" />
       <ul>
-        <li
-          v-for="(value, index) in $t('incometax.howto.list')"
-          :key="index"
-          class="mb-1 paragraph-sm xl:paragraph"
-        >
-          {{ value }}
+        <li class="mb-1 paragraph-sm xl:paragraph">
+          <i18n path="incometax.howto.options.o1.content" tag="p" class="paragraph-sm xl:paragraph">
```

Comment:
We can simplify this by just having `tag="li"` and `class="mb-1 paragraph-sm xl:paragraph"` on `i18n`  and remving `li` parent as having the `paragraph` classes on both parent and `only` child is redundant.

Reply:
Updated as proposed